### PR TITLE
print 2d array

### PR DESCRIPTION
**Pull request by Alexander-Duke**

### CODE REVIEW

File: com/company/Arrays/Arrays.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.company.Arrays;
+
+public class Arrays {
+    public static void main(String[] args) {
+        int y = 10;
+        int x = 10;
+        int[][] array = new int[y][x];
```
Comment (viacheslaev) [inline com/company/Arrays/Arrays.java]: 1. create method that will create matrix -> createMatrix(int rows, int columns), something like that

2. create method that will print matrix of any length printMatrix(......)

File: com/company/Array/Array.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package com.company.Array;
+
+import java.util.Scanner;
+import java.lang.Math;
+import static com.company.Array.constants.ArrayMessages.*;
+
+public class Array {
+    private final Scanner scanner = new Scanner(System.in);
+    private String usersInput = "";
+    private boolean isIncorrectInput;
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: redundant var

File: com/company/Array/Array.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            printMatrix(array);
+            System.out.println(RETRY_OR_FINISH_MESSAGE);
+            usersInput = scanner.nextLine();
+            usersInput = scanner.nextLine();
+        }
+    }
+
+    private void fillingTheMatrixRandomChar(char[][] array) {
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array[0].length; j++) {
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: array[i] will be more correct

File: com/company/Array/Array.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                int num = randomWithRange(33,126);
+                char charNum = (char) num;
+                array[i][j] = charNum;
+            }
+        }
+    }
+
+    private void printMatrix(char[][] array) {
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array[0].length; j++) {
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: same here

File: com/company/Array/Array.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+    private void enterTheNumberOfLines() {
+        numberOfLines = scanner.nextInt();
+    }
+
+    private void enterTheNumberOfColumns() {
+        numberOfColumns = scanner.nextInt();
+    }
+
+    int randomWithRange(int min, int max){
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: private

File: com/company/Array/Array.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: empty line

File: com/company/Array/Array.java [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+            System.out.println("");
+        }
+    }
+
+    private char[][] createMatrix() {
+        System.out.print(ENTER_THE_NUMBER_OF_COLUMNS);
+        enterTheNumberOfColumns();
+        System.out.print(ENTER_THE_NUMBER_OF_LINES);
+        enterTheNumberOfLines();
+        char[][] array = new char[numberOfLines][numberOfColumns];
```
Comment (viacheslaev) [inline com/company/Array/Array.java]: no need to create variable, just return

File: com/company/Array/Array.java [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    private void enterTheNumberOfColumns() {
+        numberOfColumns = scanner.nextInt();
+    }
+
+    int randomWithRange(int min, int max){
+
+        int range = (max - min) + 1;
+        return (int)(Math.random() * range) + min;
+    }
+}
```
Comment (viacheslaev) [inline com/company/Array/Array.java:75]: ctrl+alt+L to format code

**Review by viacheslaev — APPROVED** [review]
Good job 👍🏻

**Comment by Alexander-Duke** [comment]
2d Array any size

**Pull request closed without merging**